java/wavefrontsender.md

```diff
@@ -0,0 +1,212 @@
+
+### Step 2. Initialize the WavefrontDirectIngestionClient
+You initialize a `WavefrontDirectIngestionClient` by building it with the access information you obtained in Step 1.
+
+You can optionally call builder methods to tune the following ingestion properties:
+
+* Max queue size - Internal buffer capacity of the `WavefrontSender`. Any data in excess of this size is dropped.
+* Flush interval - Interval for flushing data from the `WavefrontSender` directly to Wavefront.
+* Batch size - Amount of data to send to Wavefront in each flush interval.
+
+Together, the batch size and flush interval control the maximum theoretical throughput of the `WavefrontSender`. You should override the defaults _only_ to set higher values.
+
+```java
+// Create a builder with the Wavefront URL and a Wavefront API token
+// that was created with direct ingestion permission.
+WavefrontDirectIngestionClient.Builder wfDirectIngestionClientBuilder =
+  new WavefrontDirectIngestionClient.Builder(wavefrontURL, token);
+
+// Optional: Override the max queue size (in data points). Default: 50,000
+wfDirectIngestionClientBuilder.maxQueueSize(100_000);
+
+// Optional: Override the batch size (in data points). Default: 10,000
+wfDirectIngestionClientBuilder.batchSize(20_000);
+
+// Optional: Override the flush interval (in seconds). Default: 1 second
+wfDirectIngestionClientBuilder.flushIntervalSeconds(2);
+
+// Create a WavefrontDirectIngestionClient.
+WavefrontSender wavefrontSender = wfDirectIngestionClientBuilder.build();
+ ```
+
+## Share a WavefrontSender
+
+Various Wavefront SDKs for Java use this library and require a `WavefrontSender` instance.
+
+If you are using multiple Wavefront Java SDKs within the same JVM process, you can instantiate the `WavefrontSender` just once and share it among the SDKs. 
+ 
+For example, the following snippet shows how to use the same `WavefrontSender` when setting up the [wavefront-opentracing-sdk-java](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java) and  [wavefront-dropwizard-metrics-sdk-java](https://github.com/wavefrontHQ/wavefront-dropwizard-metrics-sdk-java) SDKs.
+
+```java
+
+// Create a WavefrontSender
+WavefrontSender wavefrontSender = buildProxyOrDirectSender(); // pseudocode
+
+// Create a WavefrontSpanReporter for the OpenTracing SDK
+Reporter spanReporter = new WavefrontSpanReporter.Builder().
+   withSource("wavefront-tracing-example").
+   build(wavefrontSender);
+
+// Create a Wavefront reporter for the Dropwizard Metrics SDK
+MetricRegistry registry = new MetricRegistry();
+DropwizardMetricsReporter.Builder builder =   
+DropwizardMetricsReporter metricsReporter =
+   DropwizardMetricsReporter.forRegistry(registry).
+   build(wavefrontSender);
+...
+```
+
+**Note:** If you use SDKs in different JVM processes, you must instantiate one `WavefrontSender` instance per JVM process.
+
+## What's next
+Click on the Java SDK you were using to continue instrumenting your applications with Wavefront.
```

Comment:
We need clearer text instead of `Click on the Java SDK you were using to continue instrumenting your applications with Wavefront.`